src/reporting/Market.sol

```diff
@@ -445,7 +445,7 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable {
         }
 
         // Automated reporting period has not passed yet
-        if (block.timestamp < getAutomatedReportDueTimestamp()) {
+        if (!automatedReportReceived && block.timestamp < getAutomatedReportDueTimestamp()) {
```

Comment:
🤔 Out of scope of this PR, but it would probably be better if we made it so that automated dispute began (3 day timer started) as soon as the automated report was received.  That would reduce the "fastest resolution" down to just over 3 days, rather than 6 days as it is now.

Reply:
Yea I agree that makes more sense. I can do that in this PR or add a comment to do so.